content/en/docs/project-user-guide/application-workloads/deployments.md

```diff
@@ -7,4 +7,291 @@ description: 'Kubernetes Deployments'
+
+- **Environment Variables**
+
+  Supports configuring environment variables for Pod in the form of key-value pairs.
+
+  - name：Set the name of the environment variable.
+  - value：Set the value of the variable reference.
+  - type：Set the type of environment variables, support customization, configuration items, keys, and variable/variable references.
+
+  For more information about the command, please visit [Pod variable](https://kubernetes.io/docs/tasks/inject-data-application/environment-variable-expose-pod-information/?spm=a2c4g.11186623.2.20.16704b3e9qHXPb).
+
+- **Container Security Context**
+
+  A security context defines privilege and access control settings for a Pod or Container. [Pod Security Policies](https://kubernetes.io/docs/concepts/policy/pod-security-policy/).
+
+- **Sync Host Timezone**
+
+  The time zone of the container will be consistent with that of the host after synchronization.
+
+![deployments](/images/docs/project-user-guide/workloads/deployments_form_2_container_finish.png)
+
+#### Step3.2: Other Setting
+
+- **Update Strategy**
+
+   The `.spec.strategy` specifies the strategy used to replace old Pods by new ones. `.spec.strategy.type` can be "Recreate" or "RollingUpdate". "RollingUpdate" is the default value.
+
+  - **RollingUpdate (Recommended)**
+
+    A rolling update means the instance of the old version will be gradually replaced with new ones. During the upgrade process, the traffic will be load balanced and distributed to the old and new instances simultaneously, so the service will not be interrupted.
+
+  - **Recreate**
+
+    All existing Pods will be killed before new ones are created. Please note that the service will be interrupted during the update process.
+
+  For more information about the update strategy, please visit [Update-Strategies](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy).
+
+- **The number of Pods when updated**
+
+  - **The maximum unavailable number of Pods**: The maximum number of pods that can be unavailable during the update. The default value is 25%.
+  - **The maximum surge number of Pods**: The maximum number of pods that can be scheduled above the desired number of pods. The default value is 25%.
+
+{{<notice tip>}}
+
+  The number of Pods when updated in Deployment is different from StatefulSet.
+
+{{</ notice>}}
+
+- **Pod Security Context**
+
+  A security context defines privilege and access control settings for a Pod or Container. For more information about Pod Security Policies, please visit [Pod Security Policies](https://kubernetes.io/docs/concepts/policy/pod-security-policy/).
+
+- **Deployment Mode**
+
+  Select different deployment modes to change the affinity and anti-affinity of the inter-pod. In k8s, the affinity is used `podAffinity` and the anti-affinity is used `podAntiAffinity`. In the Kubesphere both affinity and anti-affinity are set `preferredDuringSchedulingIgnoredDuringExecution`.
```

Comment:
->
You can select different deployment modes to switch between inter-pod affinity and inter-pod anti-affinity. In Kubernetes, inter-pod affinity is specified as field `podAffinity` of field `affinity` while inter-pod anti-affinity is specified as field `podAntiAffinity` of field `affinity`. In KubeSphere, both `podAffinity` and `podAntiAffinity` are set to `preferredDuringSchedulingIgnoredDuringExecution`. You can enable **Edit Mode** in the top right corner to see field details.